jobs/integr8ly/ocp3/utils/polarion-reporter.yml

```diff
@@ -0,0 +1,87 @@
+---
+- job:
+    name: polarion-reporter
+    description: "The pipeline used during release testing to report test results to polarion. Doc: https://gitlab.cee.redhat.com/dbizzarr/integreatly-polarion-reporter"
+    project-type: pipeline
+    sandbox: true
+    concurrent: true
+    parameters:
+      - string:
+          name: JOB_NAME
+          description: The Jenkins job name with the Artifact
+      - string:
+          name: JOB_ID
+          description: The Jenkins job id with the Artifact
+      - string:
+          name: ARTIFACT_NAME
+          description: The name of the Artifact containing the JUnit report
+      - string:
+          name: TESTRUN_TITLE
+          description: The title to assign to the Test Run that will be created in Polarion
+      - string:
+          name: TESTRUN_TEMPLATE_ID
+          description: The template to use to create the Test Run
+    dsl: |
+        pipeline { 
+            agent { node { label "cirhos_rhel7" } }
+            options {
+                timeout(time: 120, unit: 'MINUTES')
+                ansiColor('gnome-terminal')
+                timestamps ()
+            }
+            environment {
+                POLARION = credentials('polarion')
+                POLARION_USER = "${POLARION_USR}"
+                POLARION_PASSWORD = "${POLARION_PSW}"
+            }
+            stages {
+                stage("Validate parameters") {
+                    when {
+                        expression { return !params.JOB_NAME }
+                        expression { return !params.JOB_ID }
+                        expression { return !params.ARTIFACT_NAME }
+                        expression { return !params.TESTRUN_TITLE }
+                        expression { return !params.TESTRUN_TEMPLATE_ID }
+                    }
+                    steps {
+                        error "JOB_NAME, JOB_ID, ARTIFACT_NAME, TESTRUN_TITLE and TESTRUN_TEMPLATE_ID must be defined"
+                    }
+                }
+                stage("Name") {
+                    steps {
+                        script {
+                            currentBuild.displayName = "${TESTRUN_TITLE} - ${TESTRUN_TEMPLATE_ID} - ${JOB_NAME}/${JOB_ID}"
+                        }
+                    }
+                }
+                stage("Clone polarion-reporter") {
+                    steps {
+                        git "https://gitlab.cee.redhat.com/dbizzarr/integreatly-polarion-reporter.git"
```

Comment:
Should be move this repo under the `integreatly-qe` user?

Reply:
Well if you create a new repo in `integreatly-qe` I can open a PR to it? wdyt?

Reply:
Done one better than that ;D

Reply:
Sorry what do you mean :sweat_smile:

Reply:
You should be able to create repo's now under `integreatly-qe` in gitlab :)

Reply:
@damienomurchu I've created it and I've also opened a new PR: https://gitlab.cee.redhat.com/integreatly-qe/integreatly-polarion-reporter/merge_requests/1, could you review it so that I can merge it and then I can switch the git repo in this one

Reply:
@b1zzu - approved the gitlab PR now :+1: